src/lib/viewers/archive/ArchiveViewer.js

```diff
@@ -68,11 +69,12 @@ class ArchiveViewer extends BaseViewer {
             return;
         }
 
-        /*
-            global BoxArchive
-            The BoxArchive is loaded from archive.js
-        */
-        this.archiveComponent = new BoxArchive(this.archiveEl, data);
+        try {
+            /* global BoxArchive loaded from archive.js */
+            this.archiveComponent = new BoxArchive(this.archiveEl, data);
+        } catch {
```

Comment:
Are there not more descriptive errors thrown to the catch statement?

Reply:
Can refreshing resolve the issue for the user? Are these client-side issues or data issues?

Reply:
> Can refreshing resolve the issue for the user? Are these client-side issues or data issues?

Refreshing probably cannot resolve the issue 😂. The issue maybe React issue or data issues.